### ISSUE TITLE
Multiplicative increase in basis states

### ISSUE THREAD

**Issue by ianmccul** [mptoolkit/mptoolkit#46]
Notation A^BxN means increase number of states multiplicatively from A to B.

Also, make the interval half-open so that in A^BxN or A..BxN, A is not included.

**Comment by ianmccul** [comment]
#48

**Comment by ianmccul** [comment]
Another possible notation is

A..B^N, to mean increase from (A,B] in N sweeps, multiplicatively.

**Comment by jesse-osborne** [comment]
@ianmccul Hasn’t this been implemented already? Although the comment at the beginning of mp-algorithms/stateslist.h which gives the syntax doesn’t mention multiplicative increases.

**Comment by ianmccul** [comment]
Yes, it has been implemented in [95a4443](https://github.com/mptoolkit/mptoolkit-dev/commit/95a4443e49c532219f05448e7f6d580b7407fd97), so this issue should be closed.  But it is missing some documentation: there is more description of the syntax in `void StatesList::AppendToken(char const* s)` and the comment in stateslist.h is out of date.  It needs a description on the wiki site too.  I'll do that, and then close this issue.
